contents/docs/session-replay/how-to-watch-recordings.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: How to watch recordings
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+To watch recordings, you can visit the [Replay](https://app.posthog.com/replay) page. When watching replays, you can change the playback speed, as well as select the option 'skip inactivity' - this will skip parts of the recording where the user was inactive on the page.
```

Comment:
I feel like this page would be a helpful place to do a some content on how to filter on the main Replay page as many people seem to get stuck here.